### PR TITLE
Added petition-for-expungement-from-bci pdf component (#33)

### PR DESCRIPTION
**Pull request by HappyViki**
Used acceptance-of-service as template.
Still concerned about the checkmarks; people probably aren't going to use all of them, 100% of the time:
Screenshot_2019-10-19 Petition for Expungement (Special Certificate from BCI)

### CODE REVIEW

**Comment by HappyViki** [comment]
Also: "Modified: Added petition-for-expungement-from-bci to form-bank/form-list (#33)"

This was supposed to be a separate pull request. I hope this works. I can get rid of my newer changes if it makes it easier.

File: frontend/app/forms/bank/form-list.component.js [hunk 1]
```diff
@@ -217,6 +217,19 @@ export default class FormList extends React.Component {
                 "/static/forms/reply-to-statement/03_Petitioner_Reply-3.png"
               ]}
             />
+            <FillableForm
+              name="Petition for Expungement (Special Certificate from BCI)"
+              readyForUsers={true}
+              keywords="petition for expungement special certificate from bci"
+              shortDescription=""
```
Comment (joeldenning) [inline frontend/app/forms/bank/form-list.component.js]:
```suggestion
              shortDescription="The form for petitioning for an expungement when you have a special certificate from the BCI."
```

In the app, hover over this form in the form bank to see that description present.

**Comment by joeldenning** [comment]
Thanks!

**Comment by joeldenning** [comment]
The build is failing and I don't know why. I am going to merge this and then check out if i can reproduce locally

**Pull request merged**